saleor/graphql/extensions/types.py

```diff
@@ -0,0 +1,23 @@
+import graphene
+
+from ...extensions import models
+from ..core.connection import CountableDjangoObjectType
+from .enums import ConfigurationTypeFieldEnum
+
+
+class ConfigurationItem(graphene.ObjectType):
```

Comment:
Would be nice to add `class Meta` with a `description` field that explains briefly what this object represents.